### PR TITLE
DEVOPS-7811: added logging of instance names

### PR DESCRIPTION
**Pull request by nz285**
Changed `ip_dict[instance.id]` value from a string to a tuple.  Since that function is only used in logging, nothing else needs to change.

### CODE REVIEW

File: License2Deploy/rolling_deploy.py [hunk 1]
```diff
@@ -11,6 +11,7 @@
 class RollingDeploy(object):
 
   MAX_RETRIES = 10
+  INSTANCE_NAME = "{0}-{1}{2}-{3}" #e.g. aw1-dnbi-cirrus107-qa
```
Comment (banderon1) [inline License2Deploy/rolling_deploy.py]: are you missing a hyphen between {1} and {2}? Or is the hyphen part of one of those?

File: License2Deploy/rolling_deploy.py [hunk 2]
```diff
@@ -49,6 +51,11 @@ def __init__(self,
     self.ready_wait = ready_wait
     self.health_wait = health_wait
     self.only_new_wait = only_new_wait
+  
+  def format_instance_name(self, ip):
+    region_short = 'aw1' if self.regions_conf == 'us-west-1' else 'ae1'
+    ip_end = ip.split(".")[3]
```
Comment (banderon1) [inline License2Deploy/rolling_deploy.py]: ip_end = ip.split(".")[-1] might make more sense

**Comment by banderon1** [comment]
+1, minor comment

**Comment by vmadura** [comment]
Instead of formatting instance name by amazon convention, better to fetch the name from aws .

**Pull request closed without merging**